src/GoogleFitOnFhir.Identity/Startup.cs

```diff
@@ -35,12 +39,38 @@ public override void Configure(IFunctionsHostBuilder builder)
             builder.Services.AddSingleton(sp => new GoogleFitClientContext(googleFitClientId, googleFitClientSecret, hostName));
             builder.Services.AddSingleton(sp => new StorageAccountContext(storageAccountConnectionString));
             builder.Services.AddSingleton(sp => new SecretClient(new Uri(usersKeyVaultUri), new DefaultAzureCredential()));
+            builder.Services.AddSingleton(sp =>
```

Comment:
I would like to avoid this pattern as much as possible. Rather than "manually" resolving dependencies, I would prefer to let the DI framework do all the work. See my comment about removing the 3 request classes. I think we can just pass the GoogleFitClientContext to the constructor of the GoogleFitAuthService.

I also think that the Scopes array should be a property on the GoogleFitClientContext named DefaultScopes. At some point, we will need to support dynamically adding additional scopes to the request, but we will probably always need the https://www.googleapis.com/auth/userinfo.profile scope at the very least, and it should be the default scope.